src/frontend/Komponenter/Behandling/Vurdering/EndreVurderingComponent.tsx

```diff
@@ -138,12 +148,7 @@ const EndreVurderingComponent: FC<{
                 });
             })}
             {skalViseLagreKnapp && (
-                <LagreKnapp
-                    onClick={() => {
-                        nullstillIkkePersistertKomponent(vurdering.id);
-                    }}
-                    type={'submit'}
-                >
+                <LagreKnapp onClick={håndterTrykkPåLagre} type={'submit'}>
```

Comment:
Ikke helt relatert til denne PR, men tenker det kan være fint å få fikset
Burde vi fjerne onClick her, og flytte kallet til `håndterTrykkPåLagre` til onSubmit? For få hendelser som skjer når man trykker på knappen samlet til et sted? 
Samt kanskje endre navnet på metoden til noe (kommer ikke på noe bra forslag) / eller flytte det fra submit til `håndterTrykkPåLagre` -> `lagreVurdering`

Reply:
Burde nok flyttes til `submit`, ja.